content/docs/06-integrations/00-cloudanix.md

```diff
@@ -16,35 +16,22 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Cloudanix
 
-Cloudanix container security detects threats and anomalous behaviors in kubernetes clusters. The threats and anomalies can be visualized in Cloudanix Dashboard, in real-time.
+The Cloudanix pack is an add-on security pack that provides a dashboard to help you detect threats and unusual behavior in your Kubernetes clusters.
```

Comment:
in this can you please add examples for both threats and unusual behavior. This will help doc readers to understand what type of issues does Cloudanix surface.

Reply:
added examples